styles/style.css

```diff
@@ -0,0 +1,111 @@
+body {
+    background-color: white;
+}
+
+*,
+*:before,
+*:after {
+    margin: 0;
+    padding: 0;
+}
+
+.site-info:after {
+    content: "";
+    display: table;
+    clear: both;
+}
```

Comment:
The above style is unnecessary since we are creating a utility class below, .cf, which will do the same thing once applied to the same element.